fetch.bs

```diff
@@ -1137,9 +1148,9 @@ whose <a for=request>destination</a> is "<code>audio</code>", "<code>font</code>
 "<code>object</code>" or "<code>embed</code>".
 
 <p>A <dfn export>non-subresource request</dfn> is a <a for=/>request</a>
-whose <a for=request>destination</a> is "<code>document</code>",
-"<code>report</code>", "<code>serviceworker</code>", "<code>sharedworker</code>",
-or "<code>worker</code>".
+whose <a for=request>destination</a> is "<code>audioworklet</code>", "<code>document</code>",
+"<code>paintworklet</code>", "<code>report</code>", "<code>serviceworker</code>",
+"<code>sharedworker</code>", or "<code>worker</code>".
```

Comment:
Is this what we want for worklets? This influences the way a service worker is selected for them. I'd think that we just want to reuse the service worker of the document for worklets.

cc @jakearchibald @jungkees

Reply:
Sorry I thought this required additional input from @jakearchibald & @jungkees I've updated the PR.